docs/compute-engine/api/operators.md

```diff
@@ -0,0 +1,68 @@
+# Larq Compute Engine Operators
+
+Larq Compute Engine extends the [builtin TensorFlow Lite operators](https://www.tensorflow.org/lite/guide/ops_compatibility#tensorflow_lite_operations) with optimized operators for running binarized neural networks.
+
+## Data Format
+
+Larq Compute Engine adheres to the `NHWC` data format used in TensorFlow Lite for activation tensors. In the following a tensor of format `BitTensor<int32, n>` represents a TensorFlow Lite tensor with data type `int32` containing binary values which are bitpacked across the channel dimension `n` and potentially padded with zeros. Mathematically a `0` valued bit represents a real value of `1.0` while `1` is interpreted as `-1.0`.
+
+Options for the operators are stored using [FlexBuffers](https://google.github.io/flatbuffers/flexbuffers.html) in the `.tflite` model files generated by the [Converter](/compute-engine/api/converter/) and passed as opaque values to the custom operators.
+
+## List of Operations
+
+### LceBconv2d
+
+2D binarized convolution layer.
+
+The operation will compute
+
+\\[
+y_{n,o} = \beta_o + \gamma_o \, \sigma\left(\sum_{i = 0}^{I - 1} w_{o, i} \star \mathrm{bsign}(x_{n,i})\right), \quad n \in [0, N), \, o \in [0, O)
+\\]
+
+where \\(\star\\) is the 2D cross-correlation operator, \\(N\\) is a batch size, \\(I\\) and \\(O\\) denote the number of input and output channels, and \\(\mathrm{bsign}\\) is the [binary sign function](/larq/api/math/#sign-function).
+
+**Inputs**
+
+- `Tensor<float32|int8>|BitTensor<int32,3>`: 4D input tensor \\(x\\)
+- `BitTensor<int32,3>`: 4D bitpacked binary filter tensor \\(w\\) in`OHWI` format
+- `Tensor<float32>`: Post activation multiplier \\(\gamma\\)
+- `Tensor<float32>`: Post activation bias \\(\beta\\)
```

Comment:
```suggestion
- `Tensor<float32>`: 1D post activation multiplier \\(\gamma\\)
- `Tensor<float32>`: 1D post activation bias \\(\beta\\)
```
(and maybe we should add that `filter.dim(0) == post_multiplier.dim(0) == post_bias.dim(0)` but i'm not sure how to specify that cleanly)